gitevanhsu/book-reviews
# Feature/default data

**Description**
1. Show random books on search page.
2. Save user searched keyword maximum 5 and show on page.
3. User can click keyword to search again.
4. Add app feature on homepage with RWD.
5. Add library picture on sign page.

**Test plan**
1. Reflash search page check it show different books.
2. Try search books with title.
3. Check keywords saved in local storage, and show on page.
4. Click keyword do the search as expect.
